### DISCUSSION TITLE
2.4.0: DAG fails fails while making requests.get call

### DISCUSSION THREAD

**Discussion by shubhashish9** [apache/airflow#26770] [Q&A] [answered]
2.4.0: DAG fails while fails requests.get call. It happens in both vertualenv and docker container on airflow 2.4.0
[schedulerLogs.log](https://github.com/apache/airflow/files/9671954/schedulerLogs.log)

[dag_id=s3-test-24_run_id=manual__2022-09-29T06_55_11.930070+00_00_task_id=test_request_task_24_attempt=1.log](https://github.com/apache/airflow/files/9671957/dag_id.s3-test-24_run_id.manual__2022-09-29T06_55_11.930070%2B00_00_task_id.test_request_task_24_attempt.1.log)

**Comment by potiuk** [comment, marked as answer]
Please try to avoid opening both discussion and issue - this makes a lot of noise. https://github.com/apache/airflow/issues/26769
